tracer/src/Datadog.Trace/LibDatadog/FFIVec.cs

```diff
@@ -17,10 +18,16 @@ internal readonly struct FFIVec
     internal readonly nuint Length;
     internal readonly nuint Capacity;
 
-    public byte[] ToByteArray()
+    public string ToUtf8String()
```

Comment:
I think this method is ok, and removes the additional copy, but would be glad if someone could confirm 🙂